### ISSUE TITLE
Original error was: No module named 'numpy.core._multiarray_umath'

### ISSUE THREAD

**Issue by karshan77** [numpy/numpy#13583]
Hi,

I am getting the error when I try to execute the following command:
`sudo python3.6 setup.py install`

I have installed numpy using virtualenv in ~/Downloads directory. When I tried to upgrade numpy I am getting following message:
**Requirement already up-to-date: numpy in ./envs/lib/python3.5/site-packages (1.16.3)**

My dist-package available in /usr/local directory. Will that causes the error?

Thanks.

**Comment by mattip** [comment]
The command you are running is used to build numpy from source. Unless you require special configuration, we recommend you use a binary package, preferable the one supplied by your os package manager or conda or pip. What is your goal in running `setup.py install`? In general, running commands with `sudo` should be discouraged, there are ways to install packages locally without it and using it can break your OS.

**Comment by karshan77** [comment]
Hi @mattip ,

I am trying to install object detection API as instructed in the llink: [https://pythonprogramming.net/creating-tfrecord-files-tensorflow-object-detection-api-tutorial/?completed=/custom-objects-tracking-tensorflow-object-detection-api-tutorial/](url)

I couldn't able to execute the `setup.py install` command without sudo. Because, I'm getting permission denied error.

Thanks.

**Comment by karshan77** [comment]
I'm getting the error module not found in the path **'numpy.core._multiarray_umath'**. This is the github link of numpy in core directory: [https://github.com/numpy/numpy/tree/master/numpy/core](url). No such module exist. I'm confused.

**Comment by mattip** [comment]
Ahh, I see. The command `setup.py install` is for the models repo in that tutorial. The actual error has a long message, it starts with instructions to uninstall and reinstall numpy. Did you do that? 

It is not clear to me why the tutorial reccommends `sudo pip install jupyter; sudo pip install matplotlib` rather than `sudo apt install jupyter, matplotlib` or `pip install jupyter, matplotlib`, but after that step you should repeatedly do `pip uninstall numpy` until you get an error, then `pip install numpy`. Before you do the next step, be sure your python environment works, that you can import numpy, matplotlib, jupyter, and import images and view them.

**Comment by karshan77** [comment]
Hi @mattip ,

I have tried by reinstalling numpy. It is also not working for me. I am getting the error when the setup.py file trying to install latest version of matplotlib. So, I have manually updated the matplotlib and commented out the line in the setup.py. Now, installation becomes success.

But, I am getting new when trying to train the model **"Import error: No module named 'kiwisolver'"**. Not sure why am I getting this new error. Raised a new issue.

Thanks.
